src/core/reopt.jl

```diff
@@ -66,10 +98,21 @@ end
 Solve the model using a `Scenario` or `BAUScenario`.
 """
 function run_reopt(m::JuMP.AbstractModel, s::AbstractScenario)
-	if s.site.CO2_emissions_reduction_min_fraction > 0.0 || s.site.CO2_emissions_reduction_max_fraction < 1.0
-		error("To constrain CO2 emissions reduction min or max percentages, the optimal and business as usual scenarios must be run in parallel. Use a version of run_reopt() that takes an array of two models.")
+	
+	instantiate_logger()
```

Comment:
then you lose the messages logged in the creation of s

Reply:
Good catch! I am checking before creating a new logger in the downstream function so it shouldn't be impacting any prior errors. I will double check and report back.

Reply:
Cleaned up logger instantiation to only happen in REopt.jl  when invoking the Scenario function since that step has to happen for almost all REopt runs (API and directly from REopt.jl).